myselfmarish/DynamicTeamPage
# adding logo img

## What does the PR do?
* It adds the logo image to the images folder

## How do you check this PR?
* Pull down the branch
*  See added png

## Screenshots 
None.
